### PR TITLE
Revert device type rename

### PR DESCRIPTION
**Pull request by telphan**

### CODE REVIEW

File: intel-nuc.svg [hunk 1]
```diff
@@ -1,32 +1 @@
...
-		/>
-	<path id="path18" fill="#127CC1" d="M44.189-29.458c9.155,0,12.933,4.509,12.933,11.867V69.07H88.37v-86.774
-		c0-17.628-9.386-37.084-36.74-37.084h-64.438V69.068h31.131v-98.526"/>
-	<text transform="matrix(1.0217 0 0 1 352.1602 -85.6602)" fill="#127CC1" font-family="'ArialMT'" font-size="28.8313">®</text>
-</g>
-<text transform="matrix(1 0 0 1 -94.1338 408.9209)" fill="#127CC1" font-family="'Tahoma-Bold'" font-size="218.0193">NUC</text>
-</svg>
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><path d="M16.25 6c.414 0 .75.336.75.75v9.5c0 .414-.336.75-.75.75h-9.5c-.414 0-.75-.336-.75-.75v-9.5c0-.414.336-.75.75-.75h9.5zm2.75 0c0-1.104-.896-2-2-2h-11c-1.104 0-2 .896-2 2v11c0 1.104.896 2 2 2h11c1.104 0 2-.896 2-2v-11zm-11 14v3h-1v-3h1zm4 0v3h-1v-3h1zm2 0v3h-1v-3h1zm-4 0v3h-1v-3h1zm6 0v3h-1v-3h1zm-8-20v3h-1v-3h1zm4 0v3h-1v-3h1zm2 0v3h-1v-3h1zm-4 0v3h-1v-3h1zm6 0v3h-1v-3h1zm4 15h3v1h-3v-1zm0-4h3v1h-3v-1zm0-2h3v1h-3v-1zm0 4h3v1h-3v-1zm0-6h3v1h-3v-1zm-20 8h3v1h-3v-1zm0-4h3v1h-3v-1zm0-2h3v1h-3v-1zm0 4h3v1h-3v-1zm0-6h3v1h-3v-1z"/></svg>
```
Comment (floion) [inline intel-nuc.svg]: @telphan , is this the icon that @konmouz provided?

Comment (telphan) [reply intel-nuc.svg]: @floion, yes it is

**Comment by agherzan** [comment]
Superseeded by https://github.com/resin-os/resin-intel/pull/95

**Comment by agherzan** [comment]
I'll leave the branch as it include the generic icon which we might use in the future.

**Pull request closed without merging**